python/graphstorm/run/gsgnn_lp/gsgnn_lp.py

```diff
@@ -91,8 +96,9 @@ def main(config_args):
     config = GSConfig(config_args)
     config.verify_arguments(True)
 
+    use_wg_feats = use_wholegraph(config.part_config)
     gs.initialize(ip_config=config.ip_config, backend=config.backend,
-                  use_wholegraph=use_wholegraph(config.part_config))
+                  use_wholegraph=config.use_wholegraph_sparse_emb or use_wg_feats)
```

Comment:
Shall we also update gsgnn_np.py, gsgnn_ep.py and gsgnn_node_emb.py?

Reply:
I updated gsgnn_np.py, gsgnn_ep.py and gsgnn_node_emb.py. Pls let me know if there are others.